### ISSUE TITLE
`rsync': undefined method `success?' for nil:NilClass

### ISSUE THREAD

**Issue by edwinvdgraaf** [DigitPaint/roger#34]
On line: https://github.com/DigitPaint/roger/blob/master/lib/roger/release/finalizers/rsync.rb#L68

CodeClimate coverage
https://codeclimate.com/github/DigitPaint/roger/coverage/5587d85dd86bab7e2601da4d

How is the $CHILD_STATUS set?

**Comment by flurin** [comment]
Hummm. That's supposed to be a magic global from ruby. I think the rubocop autocorrector did something to this

**Comment by edwinvdgraaf** [comment]
Aye.

https://github.com/DigitPaint/roger/commit/306a84b0fdcb6fb631c295b9a8fa3b2b7c811f16#diff-d12e35b68c9b91f475a6ab3ad7d3b30aL55

```diff
-        unless $?.success?
-          raise RuntimeError, "Rsync failed.\noutput:\n #{output}"
-        end
+        fail "Rsync failed.\noutput:\n #{output}" unless $CHILD_STATUS.success?
```

**Comment by flurin** [comment]
What ruby version does this happen? I tested it in ruby-1.9.3, ruby-2.2.0 and there doesn't seem to be an issue

**Comment by flurin** [comment]
Can you see if the tests in PR #37 branch also break for you?

**Comment by flurin** [comment]
Fixed in #37
